clients/extension/src/pages/popup/index.tsx

```diff
@@ -1,6 +1,7 @@
 import '@clients/extension/src/styles/index.scss'
 import '@clients/extension/src/pages/popup/index.scss'
 
+import { WalletCoreProvider } from '@clients/desktop/src/providers/WalletCoreProvider'
```

Comment:
not a good idea to directly refer to a provider in desktop

Reply:
I will fix it in another PR

Reply:
https://github.com/vultisig/vultisig-windows/issues/1129